src/Analysis/Ast/Impl/Analyzer/AnalysisWalker.cs

```diff
@@ -85,18 +85,12 @@ internal abstract class AnalysisWalker : PythonWalker {
         public override bool Walk(ImportStatement node) => ImportHandler.HandleImport(node);
         public override bool Walk(NonlocalStatement node) => NonLocalHandler.HandleNonLocal(node);
 
-        public override bool Walk(TryStatement node) {
-            TryExceptHandler.HandleTryExcept(node);
-            return base.Walk(node);
-        }
+        public override bool Walk(TryStatement node) => TryExceptHandler.HandleTryExcept(node);
```

Comment:
is there anyway to guard against double walking in the future?

Reply:
I'm not sure. Multiple walkers may be created and walk the same code, but perhaps AnalysisWalker should only ever see each node once...

Reply:
Maybe we could write a Test that creates a new type of Node for testing, which tracks counts calls to it's walk() function?